<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>716</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(0, 0, 0, .5);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Bebas Neue', sans-serif;
    font-size: 30px;
    font-weight: bold;
    color: #E50914;
    text-transform: uppercase;
    letter-spacing: 3px;
    text-align: center;
    display: inline-block;
    text-shadow: 
        0px -3px 0px #b20710, 
        0px -6px 0px #8b060c, 
        0px -9px 0px #660508;
    
    /* Tạo hiệu ứng cong */
    display: inline-block;
    transform: perspective(200px) rotateX(30deg) scaleY(1.5);</string>
    </property>
    <property name="text">
     <string>Netflix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nav_home">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Trang chủ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mvseries">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Phim T.hình</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mvies">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Phim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mymvies">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Danh sách của tôi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 0;
margin: 0;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/magnifying-glass-solid.svg</normaloff>../img/magnifying-glass-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nav_profile">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/user-regular.svg</normaloff>../img/user-regular.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_popular">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Mới thêm</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="main_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1211</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QWidget" name="videoWidget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>1191</width>
        <height>641</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="detail">
     <widget class="QLabel" name="lbl_detail_image">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>400</width>
        <height>225</height>
       </rect>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="detail_info_panel">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>700</width>
        <height>400</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_detail_info">
       <item>
        <widget class="QLabel" name="lbl_detail_name"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_director"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_release_date"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_genre"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_rating"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_duration"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_status"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_main_actor"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_detail_description"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_watch"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="watch">
     <widget class="QSlider" name="durationBar">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>560</y>
        <width>871</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">-webkit-appearance: none; 
  appearance: none;
  width: 100%;
  height: 8px;
  background: gray; /* Màu nền của thanh trượt */
  border-radius: 4px;
  outline: none;</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>560</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0:00:00</string>
      </property>
     </widget>
     <widget class="QSlider" name="volumeBar">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>600</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">-webkit-appearance: none; 
  appearance: none;
  width: 100%;
  height: 8px;
  background: gray; /* Màu nền của thanh trượt */
  border-radius: 4px;
  outline: none;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="video_container" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1181</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playBtn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>590</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../img/play_btn.svg</normaloff>../img/play_btn.svg</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="volumeBtn">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>600</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../img/volume-high-solid.svg</normaloff>../img/volume-high-solid.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="videoName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>601</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>timeLabel</zorder>
     <zorder>durationBar</zorder>
     <zorder>volumeBar</zorder>
     <zorder>video_container</zorder>
     <zorder>playBtn</zorder>
     <zorder>volumeBtn</zorder>
     <zorder>videoName</zorder>
    </widget>
    <widget class="QWidget" name="profile">
     <widget class="QPushButton" name="btn_avatar">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../img/arrow-up-from-bracket-solid.svg</normaloff>../img/arrow-up-from-bracket-solid.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>-150</y>
        <width>1581</width>
        <height>941</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/background.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_logout">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>570</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_avatar">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>39</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="gridWidget" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>891</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background_color : transparent;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_name">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>100</y>
            <width>601</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 7px;
padding: 5px;</string>
          </property>
          <property name="placeholderText">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_email">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>200</y>
            <width>601</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 7px;
padding: 5px;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_telephone">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>300</y>
            <width>601</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 7px;
padding: 5px;</string>
          </property>
          <property name="placeholderText">
           <string>Telephone</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>160</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>260</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="text">
           <string>Telephone</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="date_birthday">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>380</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_20">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>380</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="text">
           <string>Birthday</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_save">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>390</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>360</y>
            <width>201</width>
            <height>121</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : transparent;</string>
          </property>
          <property name="title">
           <string>Gender</string>
          </property>
          <widget class="QRadioButton" name="radio_male">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>112</width>
             <height>26</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : transparent;</string>
           </property>
           <property name="text">
            <string>Male</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radio_female">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>112</width>
             <height>26</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : transparent;</string>
           </property>
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radio_other">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>112</width>
             <height>26</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : transparent;</string>
           </property>
           <property name="text">
            <string>Other</string>
           </property>
          </widget>
          <widget class="QWidget" name="gridWidget" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>201</width>
             <height>121</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : transparent;
</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2"/>
          </widget>
          <zorder>gridWidget</zorder>
          <zorder>radio_male</zorder>
          <zorder>radio_female</zorder>
          <zorder>radio_other</zorder>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>label_2</zorder>
     <zorder>btn_avatar</zorder>
     <zorder>btn_logout</zorder>
     <zorder>lb_avatar</zorder>
     <zorder>gridWidget</zorder>
    </widget>
    <widget class="QWidget" name="favorite">
     <widget class="QWidget" name="favorite_container" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1191</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txt_search">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
